modules/src/main/modules/elasticsearch/module.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~ Hibernate Search, full-text search for your domain model
+ ~
+ ~ License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ ~ See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+  -->
+<module xmlns="urn:jboss:module:1.1" name="org.hibernate.search.elasticsearch-client" slot="${hibernate.search.version}">
+    <resources>
+        <resource-root path="rest-${elasticsearchClientVersion}.jar"/>
+        <resource-root path="sniffer-${elasticsearchClientVersion}.jar"/>
```

Comment:
Actually the property should be `elasticsearchSnifferVersion`. Or we could remove `elasticsearchSnifferVersion` from the pom if you want, but I'm not 100% sure the release cycles are synchronized.